node-repository/src/main/java/com/yahoo/vespa/hosted/provision/provisioning/Preparer.java

```diff
@@ -72,6 +72,15 @@ public Preparer(NodeRepository nodeRepository, FlagSource flagSource, Optional<H
             List<Node> accepted = groupPreparer.prepare(application, clusterGroup,
                                                         requestedNodes.fraction(wantedGroups), surplusNodes,
                                                         indices, wantedGroups);
+
+            if (requestedNodes.requireActiveParent()) {
```

Comment:
Shouldn't this be inverted? If we _not_  require active parents, we can ignore non-active parents.

Reply:
Yes, hold on

Reply:
Wait, I don't think your statement makes sense

Reply:
Something needs to be inverted here, either the name or the implementation + statement

Reply:
Changes the method name to `rejectNonActiveParents()`